### PR TITLE
Feature aero structure

### PR DESCRIPTION
**Pull request by aerialhedgehog**
This includes updates for the initial AVL wrap, as well as a process-based aerodynamics "markup" analysis object.

### CODE REVIEW

File: scripts/regression/test_aerodynamics.py [hunk 1]
```diff
@@ -214,32 +211,91 @@ def reg_values():
...
-    # --------------------------------------------------------------------  
+    # --------------------------------------------------------------------
     
-    # Cruise conditions (except Mach number)
-    conditions.freestream.mach_number = np.array([0.2]*test_num)
-    conditions.freestream.density = np.array([0.3804534]*test_num)
-    conditions.freestream.viscosity = np.array([1.43408227e-05]*test_num)
-    conditions.freestream.temperature = np.array([218.92391647]*test_num)
-    conditions.freestream.pressure = np.array([23908.73408391]*test_num)
+    # initialize the vehicle
```
Comment (aerialhedgehog) [inline scripts/regression/test_aerodynamics.py]: this block (if name==main) should stay clean, the plotting code can go in main(), just don't call plt.show() in main.

**Comment by aerialhedgehog** [comment]
[line 148 in test_aero](https://github.com/suavecode/SUAVE/pull/104/files#diff-9ad2064f36c29abb0bc4c3f1620a8773R148) --
can we shorten this line up?  either return a data dictionary, wrap the tuple over multiple lines --

```
(cd_c_r, cd_i_r, cd_m_r, 
 cd_m_fuse_base_r, cd_m_fuse_up_r, 
 cd_m_nac_base_r, ...
```

File: trunk/SUAVE/Methods/Aerodynamics/Fidelity_Zero/Drag/compressibility_drag_wing_total.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    #compute parasite drag total
+    total_compressibility_drag = 0.0
+    
+    
+    
+    ## from wings
+    for wing in wings.values():
+        
+        #parasite_drag += state.conditions.aerodynamics.drag_breakdown.parasite[wing.tag].parasite_drag_coefficient #parasite_drag_wing(conditions,configuration,wing)
+        compressibility_drag = conditions.aerodynamics.drag_breakdown.compressible[wing.tag].compressibility_drag #compressibility_drag_coefficient
```
Comment (aerialhedgehog) [inline trunk/SUAVE/Methods/Aerodynamics/Fidelity_Zero/Drag/compressibility_drag_wing_total.py]: this line can be shortened by unpacking drag_breakdown earlier

File: trunk/SUAVE/Methods/Aerodynamics/Fidelity_Zero/Drag/parasite_drag_pylon.py [hunk 3]
```diff
@@ -42,7 +50,8 @@ def parasite_drag_pylon(conditions,configuration,geometry):
 
     # Estimating pylon drag
     for propulsor in geometry.propulsors:
-        pylon_parasite_drag += pylon_factor *  conditions.aerodynamics.drag_breakdown.parasite[propulsor.tag].parasite_drag_coefficient
+        ref_area = propulsor.nacelle_diameter**2 / 4 * np.pi
+        pylon_parasite_drag += pylon_factor *  conditions.aerodynamics.drag_breakdown.parasite[propulsor.tag].parasite_drag_coefficient* (ref_area/geometry.reference_area * propulsor.number_of_engines)
```
Comment (aerialhedgehog) [inline trunk/SUAVE/Methods/Aerodynamics/Fidelity_Zero/Drag/parasite_drag_pylon.py]: let's shorten these lines too, values like reference area and number of engines can be unpacked.  in the case of number of propulsors, unpack it in the loop.

File: trunk/SUAVE/Methods/Aerodynamics/Fidelity_Zero/Drag/parasite_drag_pylon.py [hunk 4]
```diff
@@ -42,7 +50,8 @@ def parasite_drag_pylon(conditions,configuration,geometry):
...
         pylon_wetted_area   += pylon_factor *  conditions.aerodynamics.drag_breakdown.parasite[propulsor.tag].wetted_area * propulsor.number_of_engines
         pylon_cf            += conditions.aerodynamics.drag_breakdown.parasite[propulsor.tag].skin_friction_coefficient
         pylon_compr_fact    += conditions.aerodynamics.drag_breakdown.parasite[propulsor.tag].compressibility_factor
```
Comment (aerialhedgehog) [inline trunk/SUAVE/Methods/Aerodynamics/Fidelity_Zero/Drag/parasite_drag_pylon.py]: more lines to shorten

File: trunk/SUAVE/Methods/Aerodynamics/Fidelity_Zero/Drag/parasite_total.py [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+    wings = geometry.wings
+    fuselages = geometry.fuselages
+    propulsors = geometry.propulsors
+    vehicle_reference_area = geometry.reference_area
+    
+    #compute parasite drag total
+    total_parasite_drag = 0.0
+    
+    # from wings
+    for wing in wings.values():
```
Comment (aerialhedgehog) [inline trunk/SUAVE/Methods/Aerodynamics/Fidelity_Zero/Drag/parasite_total.py]: more lines to shorten

File: trunk/SUAVE/Methods/Aerodynamics/Fidelity_Zero/Drag/trim.py [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    conditions = state.conditions
+    configuration = settings
+    
+    trim_correction_factor     = configuration.trim_drag_correction_factor    
+    untrimmed_drag             = conditions.aerodynamics.drag_breakdown.untrimmed
+    
+    # trim correction
+    aircraft_total_drag_trim_corrected = trim_correction_factor * untrimmed_drag
+    
+    conditions.aerodynamics.drag_breakdown.trim_corrected_drag = aircraft_total_drag_trim_corrected
```
Comment (aerialhedgehog) [inline trunk/SUAVE/Methods/Aerodynamics/Fidelity_Zero/Drag/trim.py]: go ahead an unpack drag_breakdown to shorten this line

**Comment by aerialhedgehog** [comment]
anil as you mentioned in our email convo, test_aero_supersonic will need an update.  go ahead and take a crack at this and we'll have tim check it after quals

**Comment by anilvar** [comment]
Sure. Will try to get that script running asap.

----- Original Message -----

**Pull request merged**